contracts/http/response.go

```diff
@@ -12,6 +12,8 @@ type Response interface {
 }
 
 type ContextResponse interface {
+	// Cookie adds a cookie to the response.
+	Cookie(cookie *Cookie) ContextResponse
```

Comment:
S: Pointer is unnecessary.

```suggestion
	Cookie(cookie Cookie) ContextResponse
```

Reply:
Okay